src/App.tsx

```diff
@@ -20,17 +26,152 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    duration: 0,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleNext = (step: number) => {
+    const {
+      images, itemWidth, frameSize, duration, infinite,
+    } = this.state;
```

Comment:
```suggestion
    const {
      images, 
      itemWidth, 
      frameSize, 
      duration, 
      infinite,
    } = this.state;
```
according to previous comments, please fix this problem as well - start on a new line

Reply:
done